psamm/command.py

```diff
@@ -91,6 +92,16 @@ def init_parser(cls, parser):
     def run(self):
         """Execute command"""
 
+    def argument_error(self, msg):
+        """Raise error indicating error parsing an argument."""
+        raise CommandError(msg)
+
+    def fail(self, exc=None):
```

Comment:
But I did see the descriptions in the official website saying that when exc is not False value, if exc is a tuple, it is used; otherwise `sys.exc_info()` is called to get the exception information. Do I misunderstand this part? https://docs.python.org/2/library/logging.html